renovate.json

```diff
@@ -106,7 +106,7 @@
       "matchPackageNames": [
         "alpine"
       ],
-      "allowedVersions": "3.16"
+      "allowedVersions": "3.18"
```

Comment:
i would remove this constraint directly.
How is this not a problem anymore with #3246 ?
do we have a newer icu package in the ci automagically with this update?

Reply:
I checked it, it is a problem

Reply:
Fixed with https://github.com/ecamp/ecamp3/pull/3457/commits/593ff5551ae76fe55ce98684d36f4d9991872922